examples/helpers/eventwebhook/Example.java

```diff
@@ -0,0 +1,29 @@
+import com.sendgrid.helpers.eventsettings.EventWebhook;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.Security;
+import java.security.SignatureException;
+import java.security.spec.InvalidKeySpecException;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Example {
+
+    public static void main(String[] args) throws InvalidKeyException, NoSuchAlgorithmException,
+            NoSuchProviderException, SignatureException, InvalidKeySpecException {
```

Comment:
Are all these needed?

Reply:
yes

Reply:
Okay, I thought the `Exception` catch would take care of it.

Reply:
Oh wait yeah it should... Let me see if I can remove them since I am doing a try/catch statement.

Reply:
Even for tests I think I used to just do `throws Exception` since I don't care what exception was thrown; just that the test failed.